gym_anytrading/envs/trading_env.py

```diff
@@ -21,25 +20,28 @@ def opposite(self):
 
 class TradingEnv(gym.Env):
 
-    metadata = {'render.modes': ['human']}
+    metadata = {'render_modes': ['human']}
 
-    def __init__(self, df, window_size):
+    def __init__(self, df, window_size, render_mode=None):
         assert df.ndim == 2
 
-        self.seed()
+        assert render_mode is None or render_mode in self.metadata["render_modes"]
+        self.render_mode = render_mode
+
         self.df = df
         self.window_size = window_size
         self.prices, self.signal_features = self._process_data()
-        self.shape = (window_size, self.signal_features.shape[1])
+        self.shape = (window_size * 2, )
```

Comment:
This change seems not necessary. I was able to run the examples with a 2d shape.

Reply:
yes probably not necessary if this warning or recommendation does not matter

`utils/passive_env_checker.py:42: UserWarning: WARN: A Box observation space has an unconventional shape (neither an image, nor a 1D vector). We recommend flattening the observation to have only a 1D vector or use a custom policy to properly process the data. Actual observation shape: (30, 2)`